discovery-provider/src/queries/get_trending_playlists.py

```diff
@@ -229,98 +230,113 @@ def make_trending_cache_key(
     return f"generated-trending-playlists{version_name}:{time_range}"
 
 
-def get_trending_playlists(args, strategy):
+class GetTrendingPlaylistsArgs(TypedDict, total=False):
+    current_user_id: Optional[int]
+    with_tracks: Optional[bool]
+    time: str
+    offset: int
+    limit: int
+
+def get_trending_playlists_session(
```

Comment:
function name seems odd to me

Reply:
changed to start with underscore and end with `_with_session`
I did this for the tree spots where it I had to break it out into a session query - else the db context would be messed up.